teorija.tex

```diff
@@ -4321,6 +4320,13 @@ \subsection{Hornerjev algoritem}
   $a$ & $a_n$ & $a\krat a_n + a_{n-1}$ & $\dots$ & $\dots$ & \multicolumn{1}{|c}{$p(a)$}
 \end{tabular}
 
+Hornerjev algoritem se uporablja za:
+\begin{itemize*}
+  \item deljenje polinoma $p(x)$ z linearnim polinomom $(x-a)$
+  \item računanje vrednosti polinoma $p(x)$ v točki $a$
+  \item iskanje ničel polinoma $p(x)$
```

Comment:
Uporabljaj "polinoma `p`$, ne "polinoma `p(x)`, ker polinom je `p`, `p(x)` je pa njegova vrednost v točki `x`. Torej, ko se nanasas na vrednost v konkretni tocki napises `p(3)` ali pa `p(a)` ali kaj takega, ko pa splošno na cel polinom, potem pa `p`. To je sicer malo pikolovsko, sem že poazbil a sem bil takrat ko sem pisal tako natančen. Če se povsoj drugje piše funkcija `sin(x)` ali pa graf funkcije `f(x)` potem pusti...